pipeline/authn/authenticator_cookie_session_test.go

```diff
@@ -249,6 +250,46 @@ func TestAuthenticatorCookieSession(t *testing.T) {
 	})
 }
 
+func TestPrepareRequest(t *testing.T) {
+	t.Run("prepare request should return only configured headers", func(t *testing.T) {
+		testCases := []struct {
+			requestHeaders  []string
+			expectedHeaders []string
+			conf            *AuthenticatorForwardConfig
+		}{
+			{
+				requestHeaders:  []string{header.Authorization, header.AcceptEncoding},
+				expectedHeaders: []string{},
+				conf:            &AuthenticatorForwardConfig{},
```

Comment:
Shouldn't this be include our fallbacks? See this line:

https://github.com/ory/oathkeeper/pull/952/files#diff-0879cde6bf6394458fb28990a1d91fb15793e155257301d639d99ab6604a3601R90